lib/git/base.rb

```diff
@@ -35,56 +27,82 @@ def self.config
       return @@config ||= Config.new
     end
 
-    # initializes a git repository
-    #
-    # options:
-    #  :bare
-    #  :index
-    #  :repository
-    #
-    def self.init(working_dir, opts = {})
-      opts[:working_directory] ||= working_dir
-      opts[:repository] ||= File.join(opts[:working_directory], '.git')
+    # (see Git.init)
+    def self.init(directory, options = {})
```

Comment:
The arg `opts` was changed to `options` to be the same as `Git.init` so the `(see Git.init)` YARD directive would work.